tests/cypress/latest/e2e/unit_tests/capz_cluster.spec.ts

```diff
@@ -162,13 +166,29 @@ describe('Import CAPZ', { tags: '@full' }, () => {
   })
   );
 
+  qase(44, it('Delete the CAPZ cluster and create another from Clusterclass', () => {
+    // Delete cluster created from Fleet
+    cy.deleteCAPICluster(clusterName, timeout);
```

Comment:
Is there a reason for deleting CAPZ cluster and creating another one from ClusterClass in the same spec? Wouldn't it be better to add another spec for clusterclass?

Reply:
Here I attempted to reuse the azure identity secrets/config-maps already created in `capz_cluster.spec.ts`

Reply:
I understand that, I was simply suggesting something like the following:
```suggestion
  it('Delete the CAPZ cluster', () => {
    // Delete cluster created from Fleet
    cy.deleteCAPICluster(clusterName, timeout);
  })

  qase(44, it('Create CAPZ cluster from Clusterclass', () => {
    // Create cluster from Clusterclass UI
    cy.createCAPICluster(className, clusterName, machineName, k8sVersion, podCIDR);
```

But I think what you did might be a better option. Another option could be to create the clusterclass cluster with a different name or add the `cy.deleteCAPICluster(clusterName, timeout);` to the previous spec(qase 25), wdyt?